SPARQLWrapper/Wrapper.py

```diff
@@ -704,6 +718,10 @@ def _createRequest(self):
                 raise NotImplementedError("Expecting one of: {0}, but received: {1}".format(valid_types,
                                                                                             self.http_auth))
 
+        # The header field name is capitalized in the request.add_header method.
```

Comment:
Do we want to give preference to our internal headers or to the custom ones? Because then the order to `request.add_header()` matters.

Reply:
In this concrete development, the custom ones have more preference than the internal ones (in case somebody want to highly customize the request).

That's why it is warned in the method documentation (line 355)